tests/ttnn/nightly/unit_tests/operations/conv/test_conv2d.py

```diff
@@ -240,6 +240,7 @@ def run_conv(
         math_fidelity=math_fidelity,
         fp32_dest_acc_en=fp32_accum,
         packer_l1_acc=packer_l1_acc,
+        throttle_level=ttnn.ThrottleLevel.LEVEL_3 if slice_config else ttnn.ThrottleLevel.NO_THROTTLE,
```

Comment:
let's not add this in general, if some tests needs it it can set the throttle level